src/Instrumentation/Laravel/src/Watchers/QueryWatcher.php

```diff
@@ -45,12 +45,13 @@ public function recordQuery(QueryExecuted $query): void
 
         $attributes = [
             TraceAttributes::DB_SYSTEM => $query->connection->getDriverName(),
-            TraceAttributes::DB_NAME => $query->connection->getDatabaseName(),
-            TraceAttributes::DB_OPERATION => $operationName,
+            TraceAttributes::DB_NAMESPACE => $query->connection->getDatabaseName(),
+            TraceAttributes::DB_OPERATION_NAME => $operationName,
+            /** @phan-suppress-next-line PhanDeprecatedClassConstant */
```

Comment:
I have no idea what to do if there is no replacement

Reply:
I guess drop it is the best option